src/steamship/data/block.py

```diff
@@ -79,6 +79,11 @@ def query(
         )
         return res
 
+    def index(self, plugin_instance: Any = None):
+        """Index this block."""
+        tags = [Tag(text=self.text, file_id=self.id, block_id=self.id, kind="block")]
```

Comment:
Flagging this to make sure this is intended: 

Do we want to assign the block_id to the file_id?

Reply:
Yikes nice catch @EniasCailliau !